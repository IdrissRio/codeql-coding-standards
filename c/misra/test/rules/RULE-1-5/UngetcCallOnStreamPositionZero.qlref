rules/RULE-1-5/UngetcCallOnStreamPositionZero.ql