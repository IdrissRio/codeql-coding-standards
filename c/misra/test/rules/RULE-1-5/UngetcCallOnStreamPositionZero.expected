edges
| test.c:39:16:39:20 | *call to fopen | test.c:41:15:41:18 | *file | provenance |  |
nodes
| test.c:39:16:39:20 | *call to fopen | semmle.label | *call to fopen |
| test.c:41:15:41:18 | *file | semmle.label | *file |
subpaths
#select
| test.c:41:15:41:18 | *file | test.c:39:16:39:20 | *call to fopen | test.c:41:15:41:18 | *file | Obsolescent call to ungetc on file stream $@ at position zero. | test.c:39:16:39:20 | *call to fopen | *call to fopen |
